acsoo/addons.py

```diff
@@ -74,3 +75,32 @@ def addons_list_depends(ctx, exclude):
 
 
 addons.add_command(addons_list_depends, 'list-depends')
+
+
+@click.command()
+@click.option('--database')
+@click.option('--odoo-bin', 'odoo_bin', default=None)
+@click.option('--odoo-config', 'odoo_config',
+              type=click.Path(dir_okay=False, exists=True))
+@click.option('--git-commit', 'git_commit', is_flag=True, default=False)
+@click.option('--languages', default='')
+@click.option('--git-push', 'git_push', is_flag=True, default=False)
+@click.option('--languages', default='')
```

Comment:
default value for `--languages` should come from acsoo.cfg

Reply:
Can you add a help text explaining what the `languages` option does, it's not obvious.